### ISSUE TITLE
Software Center?

### ISSUE THREAD

**Issue by H-Tasker** [meefik/linuxdeploy#1305]
Hello.

Linux Deploy fascinates me a lot. I found instructions in my native language online. This was relatively good and yet I had extreme problems with the installation. Countless failed attempts. But now it works.

Now it's about using the system. Many different things are conceivable. Quite a few that don't need a GUI at all. Unfortunately I've only been a Windows user over the years. Therefore, use with or on the GUI is initially being considered. I'm not averse to installing tools via the terminal, unfortunately I fail with it.

I think it would be easier to install the Software Center first and then other components with its help. I found different commands on the net, which should achieve exactly this. Here, too, there were only failures.

Are there dependencies or systemic properties why this cannot work?

What do I have to do to make it? Please describe in such a way that a Windows user can understand it.

I am using the Samsung Galaxy S4 with Custom Rom, Android 7.1, arm 71/armhf 32 bit architecture. Inside is a 16GB Micro SD divided into 2 partitions. Fat32 and ext4. On the latter is the Debian Buster Linux (armhf). VNC and SSH running.

I have Ubuntu on my surfing laptop and a terminal connection can be established.

**Comment by MatejMagat305** [comment]
I will try to help you, but if I know, this apps is enable "install" system ................, you ask feature that is "in" software(in debian), unfortunately some software(like libreofice) will crash, because mobile is not common platform to use ..........., what do you want install?

**Comment by H-Tasker** [comment]
Yes. If it were that easy. The impetus for the project is the attempt to establish a connection (~VPN?!). Many circumstances are extremely inconvenient, causing normal approaches to not work (no ports, no IP, etc.). At the same time, the idea of turning the smartphone into a kind of Linux server is fascinating. Even with limited performance.

I had a Gnome "App Store" (~Software Center) installed. The program also started, but no software was included or selectable.

So everything is experimental and I'm looking for "the way".
SoftEther VPN via Azure Cloud-Connect would be great. The GUI seems to be necessary for the Azure part and unfortunately that doesn't exist for Linux.

VNC is included in the deploy. But there is a RealVNC that can also connect via the cloud. For example, that would help a lot.

**Comment by MatejMagat305** [comment]
I m sorry, I know only about ssh(which is working on port (for example) 9022 of course), but it work perfekt
